examples/examples_load_optimization/03_row_opt_example.py

```diff
@@ -0,0 +1,149 @@
+"""Example: Optimize a row of turbines
```

Comment:
This is an interesting example, but I think the plot is showing too many combinations of wind conditions and electricity values. Can this be reduced to somewhere around 50 combinations to make it easier to digest? For example, I tried limiting the wind directions to 248 and 266 degrees and think that already shows all the interesting trends we'd want to highlight.

I also think a little commentary would be helpful to explain why specific turbines are being curtailed/derated in response to the changing conditions. But maybe that's not necessary.